src/screens/Profiles/UserLibrary/components/LibraryHeader/styles.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native';
+
+const text = {
+  fontFamily: 'OpenSans',
+  fontSize: 12,
+  fontWeight: '700',
+};
+
+export const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: 15,
+  },
+  listTitleContainer: {
+    alignItems: 'center',
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+  },
+  listTitle: {
+    ...text,
```

Comment:
This is hawt. I had never thought to spread like this, and it makes amazing amounts of sense.